tests/pytests/unit/modules/test_win_servermanager.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+import salt.modules.win_servermanager as win_servermanager
+from tests.support.mock import MagicMock, patch
+
+
+@pytest.fixture
+def configure_loader_modules():
+    return {win_servermanager: {"__grains__": {"osversion": "6.2"}}}
+
+
+def test_install():
+    """
+    Test win_servermanager.install
+    """
+    mock_out = {
+        "Success": True,
+        "RestartNeeded": 1,
+        "FeatureResult": [
+            {
+                "Id": 338,
+                "Name": "XPS-Viewer",
+                "DisplayName": "XPS Viewer",
+                "Success": True,
+                "RestartNeeded": False,
+                "Message": "",
+                "SkipReason": 0,
+            }
+        ],
+        "ExitCode": 0,
+    }
+    expected = {
+        "ExitCode": 0,
+        "RestartNeeded": False,
+        "Restarted": False,
+        "Features": {
+            "XPS-Viewer": {
+                "DisplayName": "XPS Viewer",
+                "Message": "",
+                "RestartNeeded": False,
+                "SkipReason": 0,
+                "Success": True,
+            }
+        },
+        "Success": True,
+    }
+
+    mock_reboot = MagicMock(return_value=True)
+    with patch.object(
+        win_servermanager, "_pshell_json", return_value=mock_out
+    ), patch.dict(win_servermanager.__salt__, {"system.reboot": mock_reboot}):
+        result = win_servermanager.install("XPS-Viewer")
+        assert result == expected
+
+
+def test_install_restart():
+    """
+    Test win_servermanager.install when restart=True
+    """
+    mock_out = {
+        "Success": True,
+        "RestartNeeded": 1,
+        "FeatureResult": [
+            {
+                "Id": 338,
+                "Name": "XPS-Viewer",
+                "DisplayName": "XPS Viewer",
+                "Success": True,
+                "RestartNeeded": True,
+                "Message": "",
+                "SkipReason": 0,
+            }
+        ],
+        "ExitCode": 0,
+    }
```

Comment:
**suggestion (non-blocking)**: note that when the same data is found in multiple tests, especially if the same function is mocked to return that same data each time, that's an ideal place to use a pytest fixture.